src/gqlRules/gqlNameMatch/gqlNameMatch.js

```diff
@@ -0,0 +1,95 @@
+import { isGqlFile } from "../utils";
+import { isQuery, isMutation } from "../utils/isGQLOperation";
+import { relativePathToFile } from "../../utils";
+import { operationName } from "../utils/operationName";
+import { sanitizeGqlOperationText } from "../utils/sanitizeGqlOperationText";
+
+const meta = {
+  type: "problem",
+  docs: {
+    description:
+      "The Variable name GQL operation is assigned should match with the GQL operation name Ex: const Patient_Query_UserAvatar = gql` query Patient_Query_UserAvatar...`",
+    category: "gql-name-match",
```

Comment:
Would `gql-variable-name-match` be more descriptive? or do you have other suggestions for the name?

Reply:
Sounds good! Updated it